openmdao/docs/_utils/preprocess_tags.py

```diff
@@ -71,9 +74,8 @@ def make_tagindex(tagdir):
     # Once all the files exist, create a simple index.rst file
     indexfile = tagdir + "/index.rst"
 
-    for filepath, dirnames, filenames in os.walk(tagdir):
-        with open(indexfile, 'a') as index:
-            index.write("""
+    with open(indexfile, 'a') as index:
```

Comment:
Not sure why I had this inside a loop before; I only ever want to create ONE master index file.